libraries/classes/Config.php

```diff
@@ -1123,14 +1126,30 @@ public function setCookie(
                 return true;
             }
 
+            if (PHP_VERSION_ID < 70300) {
```

Comment:
```suggestion
            if (version_compare(PHP_VERSION, '7.3.0', '<=')) {
```
This would be more readable and consistent with the code base ?

Reply:
I suggested to use `PHP_VERSION_ID`. This constant requires PHP 5.2.7+, probably the other parts of the code base were added before that.

Reply:
Oh okay, good to know